include/message_data_static.h

```diff
@@ -0,0 +1,12 @@
+#ifndef MESSAGE_DATA_STATIC_H_
+#define MESSAGE_DATA_STATIC_H_
+
+#include "PR/ultratypes.h"
+
+struct MessageTableEntry{
+    /* 0x0000 */ u16 textId;
+    /* 0x0002 */ u8 typePos;
+    /* 0x0004 */ const char* segment;
```

Comment:
```suggestion
    /* 0x0 */ u16 textId;
    /* 0x2 */ u8 typePos;
    /* 0x4 */ const char* segment;
```
Leading 0's aren't needed